documentation/code-generation.md

```diff
@@ -94,3 +94,80 @@ generator –r –v –l=temp –version=v11.2.0-beta C:/azure-rest-api-specs/sp
 ## Generate Azure SDK for Go profiles
```

Comment:
in terms of cleaning up the doc, @raych1 : 
- In which scenarios a service team will want to `Generate Azure SDK for Go service packages in bulk` ? 
- Because of how processes have changed, section `Include a new package in the SDK` should be removed too (this was added 7 years ago :| )

Reply:
@tadelesh can you revisit the above instructions and update them accordingly if they are out of date?

Reply:
i suggested to remove `Generate Azure SDK for Go service packages in bulk`, `Include a new package in the SDK` and `Generate Azure SDK for Go profiles`. they are all out of date. also, it seems `Generate an Azure-SDK-for-Go service package` is out of date, but needs @jhendrixMSFT to confirm.

Reply:
Sorry I don't see this section in the diff. Is the name correct?

Reply:
It's not part of the current diff https://github.com/Azure/azure-sdk-for-go/blob/main/documentation/code-generation.md#generate-an-azure-sdk-for-go-service-package

The question is whether it should be removed in this PR or not.

Reply:
Got it. Yeah that's also out of date. Really it looks like that entire file needs to be rewritten.